src/Launcher/LauncherBase.php

```diff
@@ -55,71 +55,36 @@
 abstract class LauncherBase extends CronPlugin implements LauncherInterface {
 
   /**
-   * Default settings.
+   * {@inheritdoc}
    */
   public function defaultConfiguration() {
     return array();
   }
 
   /**
-   * Lock job.
-   *
-   * @param CronJob $job
-   *   The job to lock.
-   *
-   * @return string
-   *   Lock ID or FALSE.
+   * {@inheritdoc}
    */
-  abstract public function lock($job);
+  abstract public function lock(CronJob $job);
```

Comment:
You can remove abstract methods, it has the same effect if you define them on the interface.